server/src/main/java/io/crate/metadata/cluster/AlterTableClusterStateExecutor.java

```diff
@@ -132,15 +133,14 @@ protected ClusterState execute(ClusterState currentState, AlterTableRequest requ
                     Index[] concreteIndices = resolveIndices(currentState, request.tableIdent().indexNameOrAlias());
 
                     // These settings only apply for already existing partitions
-                    List<String> supportedSettings = TableParameters.PARTITIONED_TABLE_PARAMETER_INFO_FOR_TEMPLATE_UPDATE
+                    List<Setting> supportedSettings = TableParameters.PARTITIONED_TABLE_PARAMETER_INFO_FOR_TEMPLATE_UPDATE
                         .supportedSettings()
                         .values()
                         .stream()
-                        .map(Setting::getKey)
                         .collect(Collectors.toList());
```

Comment:
> defining in `TableParameters`

it's basically `SUPPORTED_NON_FINAL_SETTINGS_DEFAULT` so already defined - can just make it public and use it. Sounds good since we don't create extra object via `values.collect` but would break style consistency - so far we expose all settings groups via `TableParameters`. 

Since it's `DDL` I'm fine with either way, maybe @seut can add something here?

Reply:
I think it makes sense to not define it here in the executor, but as you like.